pkg/errors/errors.go

```diff
@@ -747,6 +753,12 @@ func OrganisationNotFound(externalID string, err error) *ServiceError {
 	return NewWithCause(svcErr.Code, svcErr, reason, externalID)
 }
 
+// OrganisationNameInvalid indicates that OCM organisation was found, but its name is invalid.
+func OrganisationNameInvalid(externalID string, name string) *ServiceError {
+	reason := "organisation with external id %s has invalid name %q"
+	return New(ErrorGeneral, reason, externalID, name)
```

Comment:
nit-nitpick: I would suggest removing the variable and placing the string inline. It will improve readability a bit. And I would suggest that we also quote ID to easier distinguish variables from the rest of the message.
```suggestion
	return New(ErrorGeneral, "organisation with external id %q has invalid name %q", externalID, name)
```